src/Integration/WPF/RelayCommandBase.cs

```diff
@@ -32,7 +32,7 @@ public void RequeryCanExecute()
         {
             if (System.Windows.Application.Current != null)
             {
-                Debug.Assert(System.Windows.Application.Current.Dispatcher.CheckAccess(), "RequeryCanExecute should be called from the UI thread");
+                Debug.Assert(Microsoft.VisualStudio.Shell.ThreadHelper.CheckAccess(), "RequeryCanExecute should be called from the UI thread");
```

Comment:
This class doesn't have a reference to an `IHost` object so I've used `ThreadHelper.CheckAccess()` directly here.